web-ui/src/main/resources/catalog/views/default/templates/advancedSearchForm/defaultAdvancedSearchForm.html

```diff
@@ -5,34 +5,37 @@ <h3 data-translate="">what</h3>
       <div class="form-group">
         <div class="col-sm-12">
           <label for="categoriesF"
-                class="control-label"
-                data-translate="">categories</label>
-          <input type="text" id="categoriesF" value=""
-                 class="form-control"
-                 gn-typeahead="categoriesOptions"
-                 gn-values="searchObj.params._cat"/>
+                class="control-label">
+            <span data-translate="" class="control-label-span">categories</span>
+            <input type="text" id="categoriesF" value=""
+                  class="form-control"
+                  gn-typeahead="categoriesOptions"
+                  gn-values="searchObj.params._cat"/>
+          </label>
         </div>
       </div>
       <div class="form-group">
         <div class="col-sm-12">
           <label for="keywordsF"
-                class="control-label"
-                data-translate="">keywords</label>
-          <input type="text" id="keywordsF" value=""
-                 class="form-control"
-                 gn-values="searchObj.params.keyword"
-                 gn-typeahead="keywordsOptions"/>
+                class="control-label">
+            <span data-translate="" class="control-label-span">keywords</span>
+            <input type="text" id="keywordsF" value=""
+                  class="form-control"
+                  gn-values="searchObj.params.keyword"
+                  gn-typeahead="keywordsOptions"/>
+          </label>
```

Comment:
This kind of tricks (label including input)... do we have some style guide? Should we? I wasn't aware until I saw this here.

Reply:
This is also how Bootstrap does it, but no, we don't have a styleguide. 

I can have a look